client/landing/jetpack-cloud/sections/scan/upsell.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import SecurityIcon from 'landing/jetpack-cloud/components/security-icon';
+import Main from 'components/main';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import StatsFooter from 'landing/jetpack-cloud/components/stats-footer';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+function ScanUpsellPage( props ) {
+	return (
+		<Main wideLayout className="scan__main">
+			<DocumentHead title="Scanner" />
+			<SidebarNavigation />
+			<div className="scan__content">
+				<SecurityIcon icon="info" />
+				<h1 className="scan__header">{ translate( 'Your site does not have scan' ) }</h1>
+				<p>
+					{ translate(
+						'Get automatic scanning and one-click fixes to keep your site one step ahead of security threats.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Automatic scanning and one-click fixes keep your site one step ahead of security threats.' )` *ES Score: 10*